Arena.toml

```diff
@@ -1590,6 +1770,16 @@ document = "stjudecloud/workflows:/tools/bwa.wdl"
 message = "bwa.wdl:303:13: note[TrailingComma]: item missing trailing comma"
 permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/tools/bwa.wdl/#L303"
 
+[[diagnostics]]
+document = "stjudecloud/workflows:/tools/cellranger.wdl"
+message = "cellranger.wdl:43:16: note[DisallowedInputName]: declaration identifier must be at least 3 characters"
+permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/tools/cellranger.wdl/#L43"
+
+[[diagnostics]]
+document = "stjudecloud/workflows:/tools/cellranger.wdl"
+message = "cellranger.wdl:95:14: note[DisallowedOutputName]: declaration identifier must be at least 3 characters"
+permalink = "https://github.com/stjudecloud/workflows/blob/129060215443bbd1b67c4d3d149d48acdab90ef2/tools/cellranger.wdl/#L95"
+
```

Comment:
The first is `id` which seems like it could be allowed, maybe? The second is `qc` which I think is too vague and correctly being flagged. Should we make an exception for the literal `id`? Or is `id` still too vague, and it should become `run_id` or something?

Reply:
I'm OK disallowing `id` by default. The verbose `identifier` would be acceptable. The user can also `except` `id` if they really disagree.

Reply:
Sounds ok to me then 👍